torch_geometric/data/remote_backend.py

```diff
@@ -0,0 +1,72 @@
+# This file defines a set of utilities for remote backends (backends that are
+# characterize as Tuple[FeatureStore, GraphStore]).
+from typing import Tuple, Union
+
+from torch_geometric.data.feature_store import FeatureStore
+from torch_geometric.data.graph_store import GraphStore
+from torch_geometric.typing import EdgeType, NodeType
+
+
+def num_nodes(
+    feature_store: FeatureStore,
+    graph_store: GraphStore,
+    query: Union[NodeType, EdgeType],
+) -> Union[int, Tuple[int, int]]:
+    r"""Returns the number of nodes in the node type or the number of source
+    and destination nodes in an edge type by sequentially accessing attributes
+    in the feature and graph stores that reveal this number. Returns a single
+    integer in the case of a feature store and a tuple of integers in the case
+    of a graph store."""
```

Comment:
I am confused by the part that says `Returns a single integer...`. Don't the number of integers returned depend only on the query.
```suggestion
    in the feature and graph stores that reveal this number. """
```

Reply:
Thanks for the catch, will update